Sketch.java

```diff
@@ -1,36 +1,132 @@
 import processing.core.PApplet;
-
+/**
+* A code that creates a game where there is snow falling and the player has to avoid hitting the snow
+* @author: Ethan Rodrigues
+*/
 public class Sketch extends PApplet {
-	
-	
-  /**
-   * Called once at the beginning of execution, put your size all in this method
-   */
+  // create arrays for the falling snow
+  float[] circleY = new float[10];
+  float[] circleX = new float[10];
+  boolean[] ballHideStatus = new boolean[10];
+  
+  // set speed for falling snow
+  float circleSpeed = 1;
+
+  // declare variables for the lives and location of the player
+  float playerCircleX = 200;
+  float playerCircleY = 200;
+  int intLives = 3;
+
   public void settings() {
-	// put your size call here
+    // screen size
     size(400, 400);
   }
 
-  /** 
-   * Called once at the beginning of execution.  Add initial set up
-   * values here i.e background, stroke, fill etc.
-   */
   public void setup() {
-    background(210, 255, 173);
+    // background colour
+    background(212, 244, 255);
+
+    // set locations for the snow within the height and width of the screen
+    for (int i = 0; i < circleX.length; i++){
+      circleY[i] = random(height);
+      circleX[i] = random(width);
+    }
   }
 
-  /**
-   * Called repeatedly, anything drawn to the screen goes here
-   */
   public void draw() {
-	  
-	// sample code, delete this stuff
-    stroke(128);
-    line(150, 25, 270, 350);  
+    // background colour
+    background(212, 244, 255);
 
-    stroke(255);
-    line(50, 125, 70, 50);  
+    // controls for changing the speed of the snow
+    if (keyPressed){
+      if (keyCode == UP){ 
+        circleSpeed = (float)(circleSpeed - 0.1);
```

Comment:
this causes the snow to go in the opposite direction (upward), we don't want that.  Just need to fall slower